docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3214,6 +3214,69 @@ In general, the same logic as is outlined for the census can be applied to all h
 
 For the two surveys currently outlined in the model, the default rates are 1.45% for ACS, and 29.05% for CPS.  
 
+**Duplicates:** 
+
+Duplication is most relevant for the census, which has a significant 
+amount of known duplicates. Often, the first PRL task is to match within 
+the same dataset in order to remove duplicates.  
+
+As a simplifying assumption in our initial model, we will be including 
+duplicates in the census only and limiting it to guardian-based duplication. 
```

Comment:
I'm curious about the decision not to include simple duplication with a certain rate across the whole population. It does not seem like it would be much more difficult to include here. It might be a bit too easy to de-duplicate, especially because address does not have "incorrect choose" column noise, but with the current docs anyone >24 years old would be guaranteed not to be a duplicate.

Reply:
Yes, that is definitely a limitation currently. I felt that having the most basic duplication was very easy for PRL. So then we discussed having noise parameters for duplicates only indicating how much noise they should have. But Abie felt this was complex and not needed in the simple model. 

So we ended with just having guardian based duplicates. If we want to revisit this with Abie, I am happy to add other duplication later.

Reply:
Makes sense. I agree it doesn't matter much either way since it will be quite easy to deduplicate those without cranking up the column noise on duplicates.